File: backend/src/RepoFluent.Application/ReviewDecision.cs
namespace RepoFluent.Application;

public sealed record ReviewDecision(
    string ReviewerId,
    string TenantId,
    string PackageId,
    string PackageVersion,
    string PackageChecksum,
    string ValidationIssueChecksum,
    string Decision,
    DateTimeOffset DecidedAt,
    WarningAcknowledgement? WarningAcknowledgement,
    string? Rationale);

File: frontend/projects/api/src/lib/review-decision.ts
import type { WarningAcknowledgement } from './warning-acknowledgement';

export interface ReviewDecision {
  reviewerId: string;
  tenantId: string;
  packageId: string;
  packageVersion: string;
  packageChecksum: string;
  validationIssueChecksum: string;
  decision: string;
  decidedAt: string;
  warningAcknowledgement: WarningAcknowledgement | null;
  rationale: string | null;
}
